src/migration-scripts/interfaces/33-to-34

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
I have no objections against the main part of the PR related to the missing `autoconf` option. However, I don't understand why it includes a migration script for a task about wireless interface country codes that seems completely unrelated to the issue of T7646. Is it an accidental `git add` or I'm missing something here?